### ISSUE TITLE
Colors are not respected

### ISSUE THREAD

**Issue by BachoSeven** [calculon102/xgifwallpaper#1]
Hi, I just tried the AUR package for this.
Running it without options on a random gif seems to distort the colors: (inverted?)
![img](https://user-images.githubusercontent.com/55460785/92146319-d39b8800-ee19-11ea-93bc-9a9572d19c57.png)

**Comment by calculon102** [comment]
Thank you very much for testing, @BachoSeven!

I fixed this a moment ago and publish a version v0.1.1 to [AUR](https://aur.archlinux.org/packages/xgifwallpaper/).

Can you confirm the fix is working for you?

**Comment by BachoSeven** [comment]
It does indeed. I really like your package for the following reason:

I have tried many(!) solutions/hacks to set .gifs as wallpapers, and even some of the "simpler" ones using just ffmpeg and imagemagick either take up `a lot` of CPU, or change the speed/frequency of the gifs, making it useless imo.

What I had settled on was a combination of an old UbuntuForums script using `gifsicle`[gifview]+shant-xwinwrap-bzr+my own stuff; and while this worked quite well, it still takes up about 15% of my entire CPU, or 65% of a single core, which is a lot.

Your program seems to take up much less CPU, and instead it uses more memory (about 22% on my system) which seems to be more efficient to me.

Thanks again!
